src/components/VxpPanel.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <StackLayout>
+    <StackLayout class="vxp-panel-title">
+      <VxpLabel v-if="title" class="vxp-panel-title__title" :text="title"></VxpLabel>
+      <slot name="topbar"></slot>
+    </StackLayout>
+    <StackLayout class="vxp-panel-content">
+      <slot name="content"></slot>
+    </StackLayout>
+  </StackLayout>
+</template>
+
+<script>
+import StackLayout from '../layouts/StackLayout';
+import VxpLabel from './VxpLabel';
+
+export default {
+  name: 'VxpPanel',
+  components: {
+    StackLayout,
+    VxpLabel,
+  },
+  props: {
+    title: String,
+  },
+  methods: {},
+};
+</script>
+
+<style lang="scss" scoped>
+@import './../assets/styles/helpers.scss';
+.vxp-panel-content {
+  background-color: #f4f5fa;
+  padding: unit(0) unit(15);
+}
+.vxp-panel-title {
+  display: flex;
+  height: 40px;
+  background-color: #c0cad8;
+  color: #434855;
+  box-sizing: border-box;
```

Comment:
some of the css rules used in this component are not supported by NativeScript. It means that the behavior will be different on web and native platforms.  
Could you please be sure that all css rules used in this file are supported by NativeScript?
NS Supported css rules list can be found on this link;
https://docs.nativescript.org/ui/styling#supported-css-properties

Reply:
Unsupported css are removed